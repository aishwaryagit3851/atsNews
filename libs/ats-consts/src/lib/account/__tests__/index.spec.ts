import {
  ACCOUNT_TYPES_OBJ,
  ACCOUNT_TYPES,
  DEFAULT_COMPANY_ACCOUNT,
  DEFAULT_COMPANY_ACCOUNT_WALK_IN_CUSTOMER,
  Role,
  USER_TYPES
} from '..';

describe('ATS CONSTS for - account', () => {
  it('Match consts to snapshot', () => {
    expect(DEFAULT_COMPANY_ACCOUNT).toMatchSnapshot();
    expect(DEFAULT_COMPANY_ACCOUNT_WALK_IN_CUSTOMER).toMatchSnapshot();
    expect(ACCOUNT_TYPES).toMatchSnapshot();
    expect(ACCOUNT_TYPES_OBJ).toMatchSnapshot();
    expect(Role).toMatchSnapshot();
    expect(USER_TYPES).toMatchSnapshot();
  });
});
