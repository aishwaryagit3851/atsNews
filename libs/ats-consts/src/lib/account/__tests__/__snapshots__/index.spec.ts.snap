// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - account Match consts to snapshot 1`] = `"CASH"`;

exports[`ATS CONSTS for - account Match consts to snapshot 2`] = `"Walk In Customer"`;

exports[`ATS CONSTS for - account Match consts to snapshot 3`] = `
[
  "CUSTOMER",
  "SUPPLIER",
  "EMPLOYEE",
  "CASH",
  "BANK_ACCOUNT",
]
`;

exports[`ATS CONSTS for - account Match consts to snapshot 4`] = `
{
  "BANK_ACCOUNT": "BANK_ACCOUNT",
  "CASH": "CASH",
  "CUSTOMER": "CUSTOMER",
  "EMPLOYEE": "EMPLOYEE",
  "SUPPLIER": "SUPPLIER",
}
`;

exports[`ATS CONSTS for - account Match consts to snapshot 5`] = `
{
  "ADMIN": "ADMIN",
  "CUSTOMER": "CUSTOMER",
  "EMPLOYEE": "EMPLOYEE",
  "SUPER_ADMIN": "SUPER_ADMIN",
  "SUPPLIER": "SUPPLIER",
}
`;

exports[`ATS CONSTS for - account Match consts to snapshot 6`] = `
[
  "SUPER_ADMIN",
  "ADMIN",
  "EMPLOYEE",
  "CUSTOMER",
  "SUPPLIER",
]
`;
